wordpress-mobile/WordPress-iOS
# My Site Dashboard: sync posts in the post card

Part of #17697

When showing the posts cards sync the posts.

### To test

Initial loading

1. Copy and paste the content of [this gist](https://gist.github.com/leandroalonso/9853c23769c2a33c8e41e1680438f3c1) into `BlogDetailsViewController.m`
2. Delete the app from the simulator/device
3. Run it
4. Login
5. Select a blog
6. Make sure that a loader indicator appear and the loaded posts are shown
7. Close the app

Syncing changes

1. Go on another device or use the web version
2. Update the title of one of the posts that are appearing
3. Re-run the app
4. Check that the changes are reflected

## Additional info

Due to the use of `intrinsicContentSize` you can see the content changing it's height right at the beginning. I will investigate that in a future PR.

## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
